src/aliceVision/sfmData/colorize.cpp

```diff
@@ -12,17 +12,48 @@
 #include <aliceVision/stl/mapUtils.hpp>
 #include <aliceVision/image/io.hpp>
 #include <aliceVision/system/ProgressDisplay.hpp>
+#include <aliceVision/system/Timer.hpp>
 
 #include <map>
 #include <random>
 #include <vector>
 #include <functional>
 namespace aliceVision {
+
+class LMColorAccumulator
```

Comment:
I don't know if there is a particular reason to put this in `aliceVision` instead of `aliceVision::sfmData`.

Reply:
Thought sfmData was more likely to have something name LMColor Accumulator than aliceVision. Will move it into sfmData